x/pot/types/key.go

```diff
@@ -31,7 +31,8 @@ var (
 	MatureTotalRewardKeyPrefix   = []byte{0x14} // key: prefix{address}_mature_total
 	ImmatureTotalRewardKeyPrefix = []byte{0x15} // key: prefix{address}_immature_total
 
-	EpochRewardsKeyPrefix = []byte{0x51} // key: prefix{epoch}, value: rewardDetailMap (map[node_address string]types.Reward)
+	EpochRewardsKeyPrefix     = []byte{0x51} // key: prefix{epoch}, value: []types.Reward
```

Comment:
I see now other place is using it.

Reply:
removed EpochRewardsKeyPrefix. No use any more.